### PR TITLE
Add new IMUs

### PR DESCRIPTION
**Pull request by kounocom**

### CODE REVIEW

**Review by TheButlah — APPROVED** [review]

File: protocol/typescript/src/solarxr-protocol/datatypes/hardware-info/imu-type.ts
Comment (ImUrX) [inline protocol/typescript/src/solarxr-protocol/datatypes/hardware-info/imu-type.ts:16]: these trackers are not yet in firmware code

Comment (kounocom) [reply protocol/typescript/src/solarxr-protocol/datatypes/hardware-info/imu-type.ts:16]: this pr can wait until those are merged then ig (also oops I didn't see the comment before)

**Pull request closed without merging**
